### PR TITLE
q&d fix: hide stations in UI because they no longer work

### PR DESCRIPTION
**Pull request by Horrendus**
echonest is dead :(
This commit is a quick way to no longer show stations for an upcoming 0.9 release

I would recommend keeping the stations code atm as is, maybe it can be ported easily to the new Spotify APIs

### CODE REVIEW

File: src/libtomahawk/ActionCollection.cpp [hunk 1]
```diff
@@ -120,9 +120,12 @@ ActionCollection::initActions()
     m_actionCollection[ "createPlaylist" ] = new QAction( tr( "Create Playlist" ), this );
     m_actionCollection[ "createPlaylist" ]->setShortcut( QKeySequence( "Ctrl+N" ) );
     m_actionCollection[ "createPlaylist" ]->setShortcutContext( Qt::ApplicationShortcut );
+    // echonest is dead, disable stations
+    /*
     m_actionCollection[ "createStation" ] = new QAction( tr( "Create Station" ), this );
     m_actionCollection[ "createStation" ]->setShortcut( QKeySequence( "Ctrl+S" ) );
     m_actionCollection[ "createStation" ]->setShortcutContext( Qt::ApplicationShortcut );
+    */
```
Comment (muesli) [inline src/libtomahawk/ActionCollection.cpp]: Should make sure nothing depends on this QAction or we'll segfault.

File: src/tomahawk/TomahawkWindow.cpp [hunk 2]
```diff
@@ -851,7 +852,7 @@ TomahawkWindow::audioStateChanged( AudioState newState, AudioState oldState )
     Q_UNUSED(newState);
 #else
     updatePreview();
-        
+
```
Comment (Horrendus) [inline src/tomahawk/TomahawkWindow.cpp]: non-related change, but KDevelop removed the whitespace there and as it anyways was wrong I left it in the commit :)

File: src/libtomahawk/ActionCollection.cpp (hunk 1 above)
Comment (Horrendus) [inline src/libtomahawk/ActionCollection.cpp]: The newest commit should disable all occurences where this action is retrieved

**Comment by theli-ua** [comment]
Seem to be working for me, latest commit fixed crashing.

**Pull request merged**